src/field/pokemon.ts

```diff
@@ -4191,6 +4191,9 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     this.turnData = deepCopy<PokemonTurnData>(defaultTurnData);
   }
 
+  /**
+   * Custom Sam exp formula to multiply the base exp (along with form modifiers) by level/5 +1
```

Comment:
Let's not start a witch-hunt here
```suggestion
   * Custom *legacy* exp formula to multiply the base exp (along with form modifiers) by level/5 +1
```